### DISCUSSION TITLE
My sensor is stuck in an evaluating state

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17994] [Q&A]
---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1699970562740769?thread_ts=1699970562.740769&cid=C01U954MEER).*

**Comment by alangenfeld** [comment]
> I suspect that the issue happens because I removed some runs from the UI but I don't know how to recover to a stable state

Deleting runs shouldn't cause this. The only thing I can imagine causing this is the `daemon` process crashing suddenly and not being able to write the updated `job_ticks` row to a failure/success state.

> I looked into the database schema and saw there's a job_ticks table containing the sensor ticks. If I empty that table would it solve the issue ?

Yep that should do it. Clearing the whole table will remove all the tick history, which usually isn't critical information but worth being aware of. The sensor/schedule ticks are designed to be removed over time https://docs.dagster.io/deployment/dagster-instance#data-retention

**Reply by tfrancis-wellvana** [reply]
@alangenfeld is there documentation that you can point me to on how to clear the job_ticks table? i've run into this same problem.
